app/properties/PropertyGroup/PropertyGroup.scss

```diff
@@ -4,5 +4,16 @@
 
 .property:not(:first-child) {
   position: absolute;
-  top: 18%;
+
+  &:nth-child(2) {
+    top: 18%;
+  }
+
+  &:nth-child(3) {
+    top: 36%;
+  }
+
+  &:nth-child(4) {
+    top: 54%;
```

Comment:
Actually, I'm considering removing the percentages and ratios from the properties. I think when it comes to desktop browsers we need to use adaptive design as oppose to responsive.

That'll probably also help remove some other magic numbers as well since we can just use postcss to do the calculating for us.

I'll open an ongoing css refactor issue. We can also use this as an opportunity to define a variables file to more easily stay consistent across components.